### PR TITLE
chore(ng-router-store): change angular peer dependencies to >=13.0.0

### LINKED ISSUE #19: Angular 15
### Description

Can we add angular 15 to supported peer dependencies?

### Proposed solution

Add angular 15 to supported peer dependencies

### Alternatives considered

None

### Do you want to create a pull request?

Yes

### PR DESCRIPTION
**Pull request by pwespi**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/elf-ng-router-store/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[x] Other... Please describe:
  peer dependency upate
```

## What is the current behavior?

closes https://github.com/ngneat/elf-ng-router-store/issues/19

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: libs/ng-router-store/package.json [hunk 1]
```diff
@@ -21,9 +21,9 @@
   ],
   "license": "MIT",
   "peerDependencies": {
-    "@angular/common": "^13.0.0 || ^14.0.0",
-    "@angular/core": "^13.0.0 || ^14.0.0",
-    "@angular/router": "^13.0.0 || ^14.0.0",
+    "@angular/common": "^13.0.0 || ^14.0.0 || ^15.0.0",
```
Comment (NetanelBasal) [inline libs/ng-router-store/package.json:24]: I'd change it to greater than 13

Comment (pwespi) [reply libs/ng-router-store/package.json]: Good idea. Thanks.

**Comment by pwespi** [comment]
@NetanelBasal any chance you could take another look? Thanks

**Comment by pwespi** [comment]
@NetanelBasal  Thank you for the merge. A new release would also be highly appreciated.

Thanks for your great work!

**Comment by NetanelBasal** [comment]
Done

**Pull request merged**